custom/templates/DefaultRevamp/js/core/user.js

```diff
@@ -7,9 +7,9 @@ if (loggedIn == 1) {
 	var countPms = 0;
 	var countAlerts = 0;
 
-    function updateAlerts(data) {
+	function updateAlerts(data) {
```

Comment:
Did you mean to use tabs here? Nameless PHP code should be using 4 spaces. Everything else looks good.

Reply:
Well blame eslint, should I make it a 4 space?